CompilerSource/ipc/GameInformation.fbs

```diff
@@ -0,0 +1,18 @@
+
+table GameInformation {
+  background_color:int;
+  embed_game_window:bool;
+  form_caption:string;
+  left:int;
+  top:int;
+  width:int;
+  height:int;
+  show_border:bool;
+  allow_resize:bool;
+  stay_on_top:bool;
+  pause_game:bool;
+
+  text:string;
+}
+
+root_type GameInformation;

```

Comment:
Do we maybe want to consider dropping this resource altogether? I don't know how well it's used. GMS 2 has dropped it and replaced it with an instantiatable "Notes" resource which is plain text files. RTF again was a Windowsy thing. I don't use this feature anymore myself and wouldn't even use it for a quick and dirty game prototype. That said, if we drop it, I'd be happy to look into something like Markdown editing in the future since that's pretty standard now. We can also defer dropping this until later.

This thing also used to have a constructor with some useful defaults:
https://github.com/enigma-dev/enigma-dev/blob/a79faf94c27a7db0f17dc9e236c5116f08e7e738/CompilerSource/backend/resources/GameInformation.h#L30
I think that's the only resource in the original backend with defaults compiled into it too, which I found interesting. Not sure where to go with those.